api/swagger-spec/v1beta1.json

```diff
@@ -103,18 +135,50 @@
     "description": "API at /api/v1beta1 version v1beta1",
     "operations": [
      {
-      "type": "v1beta1.EndpointsList",
+      "type": "*json.watchEvent",
```

Comment:
Looks like fallout from @smarterclayton's watch changes.

1. watchEvent looks like a backdoor unversioned API type (from watch/json/types.go)  that's not supposed to be public 
2. I'm not sure the `*` syntax works -- it may well break swagger validation, which currently passes
3. We've lost documentation of the watch payload type